common/src/main/java/whocraft/tardis_refined/common/tardis/manager/TardisControlManager.java

```diff
@@ -225,8 +225,12 @@ private TardisNavLocation scanUpForPosition(TardisNavLocation startingLocation)
         return null;
     }
 
-    public void beginFlight() {
-        if (isInFlight || ticksTakingOff > 0) {return;}
+    public void beginFlight(boolean autoLand) {
```

Comment:
Do we want this method to be a boolean return type since it is constructed this way? Plus it may be useful in determining if it took off successfully, if we add a feature in the future that works with taking off and landing.

Reply:
Sounds great, added that in.